interfaces/builtin/custom_device.go

```diff
@@ -155,9 +155,17 @@ func (iface *customDeviceInterface) validateUDevTaggingRule(rule map[string]inte
 			err = iface.validateUDevValue(value)
 			if err == nil {
 				deviceName := value.(string)
-				// furthermore, the kernel name must match the name of one of
-				// the given devices
-				if !strutil.ListContains(devices, "/dev/"+deviceName) {
+				// furthermore, the kernel name must match the basename of one
+				// of the given devices
+				foundMatch := false
+				for _, devicePath := range devices {
+					deviceBasename := devicePath[strings.LastIndex(devicePath, "/")+1:]
```

Comment:
Go has a helper for this so you can just write: `deviceBasename := filepath.Base(devicePath)` which is slightly shorter.

Reply:
I was thinking to avoid another import for just one line, but I can use that instead, no problem. Thanks!

Reply:
"I was thinking to avoid another import" we never take that route with things offered by the stdlib